src/main/groovy/org/edx/jenkins/dsl/JenkinsPublicConstants.groovy

```diff
@@ -139,7 +139,14 @@ class JenkinsPublicConstants {
             permission('hudson.model.Item.Cancel', 'edx')
         }
 
+    }
 
+    public static final Closure GENERAL_PRIVATE_JOB_SECURITY = {
```

Comment:
Is this so that other job definitions can use this snippet also? Nice.

Reply:
Right. Trying to make these more DRY as I go